ext/node/polyfills/_events.mjs

```diff
@@ -469,7 +471,7 @@ function _addListener(target, type, listener, prepend) {
       w.emitter = target;
       w.type = type;
       w.count = existing.length;
-      nodeGlobals.process.emitWarning(w);
+      globalThis.process.emitWarning(w);
```

Comment:
Not too familiar with the way we branch globals: Is `globalThis` the same as `nodeGlobals` here?